js/about/preferences.js

```diff
@@ -435,27 +436,25 @@ class SecurityTab extends ImmutableComponent {
         <SettingCheckbox dataL10nId='allSiteCookies' prefKey={settings.SHUTDOWN_CLEAR_ALL_SITE_COOKIES} settings={this.props.settings} onChangeSetting={this.props.onChangeSetting} />
       </SettingsList>
       <div className='sectionTitle' data-l10n-id='passwordsAndForms' />
-      <SettingsList dataL10nId='passwordManager'>
-        <SettingCheckbox dataL10nId='usePasswordManager' prefKey={settings.PASSWORD_MANAGER_ENABLED} settings={this.props.settings} onChangeSetting={this.props.onChangeSetting}
-          options={
-            getSetting(settings.PASSWORD_MANAGER_ENABLED, this.props.settings)
-              ? <span className='linkText' data-l10n-id='managePasswords'
-                onClick={aboutActions.newFrame.bind(null, {
-                  location: 'about:passwords'
-                }, true)}></span>
-              : null
-          } />
-        <SettingCheckbox dataL10nId='useOnePassword' prefKey={settings.ONE_PASSWORD_ENABLED} settings={this.props.settings} onChangeSetting={this.props.onChangeSetting} />
-        <SettingCheckbox dataL10nId='useLastPass' prefKey={settings.LAST_PASS_ENABLED} settings={this.props.settings} onChangeSetting={this.props.onChangeSetting}
-          options={
-            getSetting(settings.LAST_PASS_ENABLED, this.props.settings)
-              ? <span className='linkText' data-l10n-id='preferences'
-                onClick={aboutActions.newFrame.bind(null, {
-                  location: 'chrome-extension://hdokiejnpimakedhajhdlcegeplioahd/tabDialog.html?dialog=preferences&cmd=open'
-                }, true)}></span>
-              : null
-          } />
-        <SettingCheckbox dataL10nId='useDashlane' prefKey={settings.DASHLANE_ENABLED} settings={this.props.settings} onChangeSetting={this.props.onChangeSetting} />
+      <SettingsList>
+        <SettingItem dataL10nId='passwordManager'>
+          <select value={getSetting(settings.ACTIVE_PASSWORD_MANAGER, this.props.settings)} onChange={changeSetting.bind(null, this.props.onChangeSetting, settings.ACTIVE_PASSWORD_MANAGER)} >
+            <option data-l10n-id='builtInPasswordManager' value={passwordManagers.BUILT_IN} />
+            <option data-l10n-id='onePassword' value={passwordManagers.ONE_PASSWORD} />
+            <option data-l10n-id='dashlane' value={passwordManagers.DASHLANE} />
+            <option data-l10n-id='lastPass' value={passwordManagers.LAST_PASS} />
+            <option data-l10n-id='doNotManageMyPasswords' value={passwordManagers.UNMANAGED} />
+          </select>
+        </SettingItem>
+        {
+          getSetting(settings.ACTIVE_PASSWORD_MANAGER, this.props.settings) === passwordManagers.BUILT_IN
+          ? <label className='linkTextSmall' data-l10n-id='managePasswords'
```

Comment:
Last pass used to have a preferences link that I think got removed in this by accident. Pls re-add